src/utils/object.ts

```diff
@@ -216,6 +216,56 @@ export function camelizeObjectKeys(obj: { [key: string]: any }) {
   return newObj;
 }
 
+/**
+ * camelizeKeys recursively turns object keys camelCased, works on
+ *   arrays of objects, and object-key's array values, which may
+ *   contain column names
+ * This does not replace camelizedObjectKeys because of its
+ *   aggressive (recursive) nature, which is a little more greedy
+ *
+ * @param arg <Array|object|string>
+ * @returns An object with camelized keys or camelized string
+ */
+export function camelizeKeys(arg: any) {
+  let response: any;
```

Comment:
avoid using `any`. Consider using a stricter type.
Avoid using `let` too. It's better to declare a variable in each if and have each if return that variable at the end of its block